### PR TITLE
Add sensor_locations

### PR DESCRIPTION
**Pull request by mhauru**

### CODE REVIEW

File: dtbase/tests/test_sensor_locations.py
Comment (mastoffel) [inline dtbase/tests/test_sensor_locations.py:1]: Could think about splitting up into test_insert_sensor_location and test_get_sensor_location tests.

File: dtbase/tests/test_api_sensors.py
Comment (mastoffel) [inline dtbase/tests/test_api_sensors.py:1]: Test could be split into insertion / retrieval tests too.

**Review by mastoffel — COMMENTED** [review]
Great job Markus, everything runs well. Just a few minor comments.

File: dtbase/tests/test_api_sensors.py
Comment (mhauru) [reply dtbase/tests/test_api_sensors.py:1]: Done

File: dtbase/tests/test_sensor_locations.py
Comment (mhauru) [reply dtbase/tests/test_sensor_locations.py:1]: Done

File: dtbase/tests/test_sensor_locations.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "latitude": test_locations.LATITUDE1,
+        "longitude": test_locations.LONGITUDE1,
+    }
+    date1 = dt.datetime.now() - dt.timedelta(days=2)
+    sensor_locations.insert_sensor_location(
+        uniq_id, "latlong", coordinates1, date1, session=session
+    )
+    locations = sensor_locations.get_location_history(uniq_id, session=session)
+    assert len(locations) == 1
+    print([locations[0][k] == v for k, v in coordinates1.items()])
```
[the remark this answers is not in the document]
Comment (mhauru) [reply dtbase/tests/test_sensor_locations.py]: Yup, done.

**Pull request merged**